syncall/tw_caldav_utils.py

```diff
@@ -64,9 +64,20 @@ def convert_tw_to_caldav(tw_item: Item) -> Item:
         caldav_item["start"] = tw_item["due"] - timedelta(hours=1)
         caldav_item["due"] = tw_item["due"]
 
+    # Tags
     if "tags" in tw_item.keys():
         caldav_item["categories"] = tw_item["tags"]
 
+    # Project
+    if "project" in tw_item.keys():
+        # If there are tags use them to prevent overwritting
+        if "tags" in tw_item.keys():
+            project = tw_item["tags"]
```

Comment:
Do you need to do this explicitly? How about initializing `project` to an empty list before the ifs, then doing something like `project.extend(tw_item.get("tags", []))`